src/admin.rs

```diff
@@ -1,13 +1,22 @@
+use crate::SharedFluvio;
```

Comment:
Dang, I feel like a lot of the work you did with the Admin API here should have been done in the Rust API first, sorry if that was a blocker for you.

Reply:
Yeah, some of the comments made above are better made in the Rust client. I'll add more responses to each of the comments. Not all of these proposed changes are going to land in this PR, just an FYI. so we may need to open some issues.

Reply:
I feel like one thing that is worthwhile is changing `SharedClient` to `Arc<Fluvio>` rather than `Arc<RwLock<Fluvio>>`

Reply:
See previous comment.   `Fluvio` is sharable, just need to have clone `derive` attribute.